### DISCUSSION TITLE
No IP address from DHCP server at console

### DISCUSSION THREAD

**Discussion by miphip** [siderolabs/talos#5095] [Q&A] [answered]
Hi, I just created a bootable USB memory with talos 0.14.0 but I don't see talos receiving an IP address from DHCP. I only see "waiting for network address". When I look into the router it doesn't list the machine.

When booting into ubuntu on the same machine it does get an address by DHCP and I can see it in the router.

What can I do?

**Comment by smira** [comment, marked as answer]
Hi @miphip it's not clear what the problem might be, but I would also check if your network card is detected by Talos Linux kernel. 

You might need to find the network card driver/hardware from Ubuntu, and we can see if it's enabled with Talos Linux kernel.

**Reply by miphip** [reply]
It's an "Intel Corporation Ethernet Controller I225-V (rev 03)". It says "2.5Gb Ethernet" in the motherboard manual.
And the driver is "igc".

**Reply by miphip** [reply]
It's difficult to see the boot log at the console because I can't pause it and it spews out messages of not reaching dns at 8.8.8.8 at the end.

**Reply by smira** [reply]
Talos has `CONFIG_IGC`, but it was just recently added. Would you mind trying out 1.0.0-beta.0 version from Releases?

**Reply by miphip** [reply]
Yes, success. Thank you.
